wagtail/tests/modeladmintest/models.py

```diff
@@ -9,6 +9,11 @@ class Author(models.Model):
     name = models.CharField(max_length=255)
     date_of_birth = models.DateField()
 
+    def author_birth_string(self):
+        return '{} was born in pallet town'.format(self.name)
```

Comment:
could we lose the `self.name` from this string? This way we don't have to change the other test to make this work :)